source/_docs/configuration/templating.markdown

```diff
@@ -617,8 +617,13 @@ For example, if you wanted to select a field from `trigger` in an automation bas
 - `as_timestamp(value, default)` converts datetime object or string to UNIX timestamp. If that fails, returns the `default` value, or if omitted raises an error. This function can also be used as a filter.
 - `as_local()` converts datetime object to local time. This function can also be used as a filter.
 - `strptime(string, format, default)` parses a string based on a [format](https://docs.python.org/3.10/library/datetime.html#strftime-and-strptime-behavior) and returns a datetime object. If that fails, it returns the `default` value or, if omitted, raises an error.
-- `relative_time` converts datetime object to its human-friendly "age" string. The age can be in second, minute, hour, day, month or year (but only the biggest unit is considered, e.g., if it's 2 days and 3 hours, "2 days" will be returned). Note that it only works for dates _in the past_.
-  - Using `relative_time()` will cause templates to be refreshed at the start of every new minute.
+- `relative_time()`: deprecated in favor of time_since(). relative_time(datetime) is equivalent to time_since(datetime)
```

Comment:
We can just remove it from the documentation. We should not say it's equivalent to `time_since(datetime)` since it does not behave the same way if the datetime is in the future.
```suggestion
```